examples/certificate-check-volume.yaml

```diff
@@ -0,0 +1,48 @@
+# An example Job to run a certificate check of OpenShift's internal
+# certificate status from within OpenShift.
+#
+# It uses the openshift/openshift-ansible container image.
+# (see README_CONTAINER_IMAGE.md in the top level dir for more details).
+#
+# It expects:
+#   - the Ansible inventory file from the 'hosts' key of a ConfigMap
+#     named 'inventory'
+#   - the ssh key in the 'ssh-privatekey' key of a Secret named 'sshkey'
+#   - a PersistentVolumeClaim named 'certcheck-reports' where the
+#     generated reports are stored
```

Comment:
This header comment in this file and the one in `examples/certificate-check-upload.yaml` look very similar, if not the same in terms of content, but with an apparent unnecessary difference in formatting?